scripts/admin/elektra-install-config-files

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# @author Dominic Jäger <e1634025@student.tuwien.ac.at>
+# @brief Installs or merges a configuration file in Elektra
+# @date 2019-08-20
+# @tags creator
+
+# This script installs or merges configuration files from the file system into
```

Comment:
This should be in the man page, not here (doc/help/)

Reply:
You mean the description, not the @author, @brief?

Reply:
The tags should be here but the longer descriptions should be moved to the man page (it starts where I put the comment).